frontend/src/components/TitleBar/Settings/SettingsButton.tsx

```diff
@@ -17,7 +17,7 @@ export default function SettingsButton(): ReactElement {
       {showSettings && (
         <div className={styles.SettingsModal}>
           <button className={styles.CloseButton} type="submit" title="Close Settings" onClick={closeModal} tabIndex={-1}>
-            <SamwiseIcon iconName="x-light" />
+            <SamwiseIcon iconName="x-light-settings"/>
```

Comment:
The linter wants a space before the closing />

Reply:
ok, just committed now